src/ILCompiler.ReadyToRun/src/Compiler/ReadyToRunMetadataFieldLayoutAlgorithm.cs

```diff
@@ -15,6 +15,7 @@
 using Internal.TypeSystem;
 using Internal.TypeSystem.Ecma;
 using Internal.TypeSystem.Interop;
+using ILCompiler.DependencyAnalysis;
```

Comment:
Do we need this?

Reply:
Removed in 5th commit; thanks for noticing!